concordium-node/src/configuration.rs

```diff
@@ -158,109 +188,146 @@ pub struct BakerConfig {
         long = "heap-profiling",
         help = "Profile the heap [(`cost`,-hc), (`type`, -hy), (`module`, -hm), (`description`, \
                 -hd)] in the Haskell subsystem",
-        default_value = "none"
+        default_value = "none",
+        env = "CONCORDIUM_NODE_BAKER_HEAP_PROFILING"
     )]
     pub heap_profiling: String,
     #[cfg(feature = "profiling")]
-    #[structopt(long = "time-profiling", help = "Profile the time in the Haskell subsystem")]
+    #[structopt(
+        long = "time-profiling",
+        help = "Profile the time in the Haskell subsystem",
+        env = "CONCORDIUM_NODE_BAKER_TIME_PROFILING"
+    )]
     pub time_profiling: bool,
     #[cfg(feature = "profiling")]
     #[structopt(
         long = "backtraces",
-        help = "Show bactraces generated by exceptions in the Haskell subsystem"
+        help = "Show bactraces generated by exceptions in the Haskell subsystem",
+        env = "CONCORDIUM_NODE_BAKER_BACKTRACES"
     )]
     pub backtraces_profiling: bool,
     #[cfg(feature = "profiling")]
     #[structopt(
         long = "stack-profiling",
         help = "Include memory occupied by threads in the heap profile. Only has effect if \
-                `heap-profiling` is enabled."
+                `heap-profiling` is enabled.",
+        env = "CONCORDIUM_NODE_BAKER_STACK_PROFILING"
     )]
     pub stack_profiling: bool,
     #[cfg(feature = "profiling")]
     #[structopt(
         long = "profiling-sampling-interval",
         help = "Profile sampling interval in seconds",
-        default_value = "0.1"
+        default_value = "0.1",
+        env = "CONCORDIUM_NODE_BAKER_PROFILING_SAMPLING_INTERVAL"
```

Comment:
`CONCORDIUM_NODE_PROFILING_SAMPLING_INTERVAL`

Reply:
Shouldn't this be in the "Runtime" group as well?